src/dataregistry/registrar.py

```diff
@@ -337,7 +337,6 @@ def register_dataset(
             owner_type = "user"
         if owner_type not in ["user", "group", "production"]:
```

Comment:
This needs to be ["user", "group", "project", "production"] to do the right thing now.
But really I would prefer something more formal.   The `Enum` in the database definition was awkward and unnecessary since the value for owner_type is being checked in the code, but there should be something - could be a Python enum definition or just a set defined in Registrar.py (and made public), e.g.
```OWNER_TYPES = {"user", "project", "group", "production"}```
That way the user can find out what allowed values are, and the definition of what's allowed is not buried in code, making it easier to maintain.

Reply:
I've bumped the DB version, and added a set of `OWNER_TYPES` at the top of `register.py` as suggested.

I've also updated to the docs mentioning that you can pass `owner` and `owner_type` during initiation to set them universally,